skipper.go

```diff
@@ -476,7 +479,7 @@ func createDataClients(o Options, auth innkeeper.Authentication) ([]routing.Data
 
 		clients = append(clients, etcdClient)
 	}
-
+    
```

Comment:
please remove this whitespace.
You can also run "gofmt -sw" on this file

Reply:
see https://travis-ci.org/zalando/skipper/builds/385732378?utm_source=github_status&utm_medium=notification

Reply:
or `make fmt`